frontend/src/map/Tile.js

```diff
@@ -1,9 +1,18 @@
+const colors = {
+    grass: 0x12b464,
+    dirt: 0xce8b55,
+    stone: 0x928e85,
+    unknown: 0xc23be8,
+}
+
 export class TileType {
     static Grass = new TileType('grass')
     static Dirt = new TileType('dirt')
+    static Stone = new TileType('stone')
 
     constructor(name) {
         this.name = name
+        this.color = colors[name]
```

Comment:
Should we specify the default here as well? Something like:
```suggestion
        this.color = colors[name] ?? colors['unknown']
```